packages/components/resource-pages/data-resource-details-page.tsx

```diff
@@ -103,6 +103,9 @@ const DataResourceDetailsPage: React.FC<DataResourceDetailsPageProps> = ({
         customKebabItems={(t) => ({
           Delete: {
             value: t('Delete data source'),
+            props: {
+              'data-test-dropdown-menu': 'delete-data-source',
+            },
```

Comment:
i don't think this will be required... check bucket policy PR one, I added there few `data-test`, we can use those...

Reply:
this is for each drop-down item, not the dropdown as a while.

Reply:
check the PR... deletion `it` block...

Reply:
@debjyoti-pandit  Does this mean that every item is sharing the same `data-test` ?

Reply:
no this is just the button on  which we click and the dropdown expands